core/src/main/java/org/mule/runtime/core/el/v2/MuleExpressionLanguage.java

```diff
@@ -74,4 +87,17 @@ public ValidationResult validate(String expression) {
     return expressionExecutor.validate(expression);
   }
 
+  private void addEventBindings(MuleEvent event, BindingContext.Builder contextBuilder) {
+    Message message = event.getMessage();
+    Attributes attributes = message.getAttributes();
```

Comment:
Attributes should be TypedValue in Message. Ensure there is a TODO and JIRA issue.